app/controllers/vendor_api/interviews_controller.rb

```diff
@@ -3,12 +3,17 @@ class InterviewsController < VendorAPIController
     include ApplicationDataConcerns
     include APIValidationsAndErrorHandling
 
+    rescue_from InterviewWorkflowConstraints::WorkflowError, with: :render_workflow_error
+    rescue_from NotModifiedError, with: :render_not_modified
+    rescue_from InvalidProviderCode, with: :render_invalid_provider_code
```

Comment:
can we add `InvalidProviderCode` as an error as rails now fails to load?

Reply:
added